pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/guide/pages/SearchItemsAndFluidsPage.kt

```diff
@@ -0,0 +1,55 @@
+package io.github.pylonmc.pylon.core.guide.pages
+
+import io.github.pylonmc.pylon.core.guide.PylonGuide
+import io.github.pylonmc.pylon.core.guide.button.FluidButton
+import io.github.pylonmc.pylon.core.guide.button.ItemButton
+import io.github.pylonmc.pylon.core.guide.pages.base.SearchPage
+import io.github.pylonmc.pylon.core.i18n.AddonTranslator
+import io.github.pylonmc.pylon.core.registry.PylonRegistry
+import io.github.pylonmc.pylon.core.util.key.getAddon
+import io.github.pylonmc.pylon.core.util.pylonKey
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer
+import org.bukkit.Material
+import org.bukkit.entity.Player
+import xyz.xenondevs.invui.item.Item
+
+class SearchItemsAndFluidsPage : SearchPage(
+    pylonKey("search"),
+    Material.OAK_SIGN
+) {
+
+    fun getItemButtons(player: Player): MutableList<Pair<Item, String>> = PylonRegistry.ITEMS.getKeys().filter {
+        !PylonGuide.hiddenItems.contains(it)
+    }.map {
+        val translator = AddonTranslator.translators[getAddon(it)]!!
+        val name = translator.translate(
+            Component.translatable("pylon.${it.namespace}.item.${it.key}.name"), player.locale()
+        )
+        check(name != null)
+        val plainTextName = serializer.serialize(name).lowercase()
+        Pair(ItemButton(it), plainTextName)
+    }.toMutableList()
```

Comment:
```suggestion
    fun getItemButtons(player: Player): MutableList<Pair<Item, String>> = PylonRegistry.ITEMS.getKeys()
    .filter { !PylonGuide.hiddenItems.contains(it) }
    .map {
        val translator = AddonTranslator.translators[getAddon(it)]!!
        val name = translator.translate(
            Component.translatable("pylon.${it.namespace}.item.${it.key}.name"), player.locale()
        )
        check(name != null)
        val plainTextName = serializer.serialize(name).lowercase()
        Pair(ItemButton(it), plainTextName)
    }
    .toMutableList()
```
My preferred formatting, feel free to ignore